docs/02-for-app-authors/03-metainfo-guidelines/01-quality-guidelines.md

```diff
@@ -382,11 +382,18 @@ remove the shadow and rounding.
 ### Reasonable window size
 
 In order to ensure that text and interface elements are visible scaled
-down in the app store UI the window size should be 1000x700 pixels or
-smaller (2000x1400 for HiDPI).
+down in the app store UI, the window size should typically be around 
```

Comment:
you didn't touch this but "are visible scaled down" seems like wrong grammar. What is this trying to say?

"are visibly scaled down" or "are visible and scaled down"?